package/yast2-users.changes

```diff
@@ -2,10 +2,22 @@
 Mon Nov  9 10:52:39 UTC 2015 - igonzalezsosa@suse.com
 
 - fix validation of AutoYaST profiles (bsc#805275)
-- 3.1.35.1
+- 3.1.37.1
 
 -------------------------------------------------------------------
-Thu Nov 20 10:44:41 CET 2014 - jsuchome@suse.cz
```

Comment:
I tried to keep in sync with the changes file which is already published in IBS: https://build.suse.de/package/view_file/SUSE:SLE-12:Update/yast2-users/yast2-users.changes?expand=1

Should I change it anyway?